CommunityProject/Assets/_Game/Scripts/Runtime/BuildSystem/BuildController.cs

```diff
@@ -104,7 +114,7 @@ private void ClearBuildContext()
 		private void ReadBuild(Vector2 position)
 		{
 			// We ignore the position parameter here, so we do not have to raycast for the correct position again.
-			if (_buildContext is { IsValidPosition: false })
+			if (_buildContext is null or { IsValidPosition: false })
```

Comment:
Doppelter Null Check.
```suggestion
			if (_buildContext is null or _buildContext.IsValidPosition: false)
```